requirements-devel.txt

```diff
@@ -27,8 +27,6 @@
 #
 #     -e git+git://github.com/mitsuhiko/werkzeug.git#egg=Werkzeug
 #     -e git+git://github.com/mitsuhiko/jinja2.git#egg=Jinja2
-
--e git+git://github.com/inveniosoftware/invenio-base.git#egg=invenio-base
--e git+git://github.com/inveniosoftware/invenio-ext.git#egg=invenio-ext
--e git+git://github.com/inveniosoftware/invenio-upgrader.git#egg=invenio-upgrader
--e git+git://github.com/inveniosoftware/invenio-utils.git#egg=invenio-utils
+-e git+git://github.com/inveniosoftware/invenio-db#egg=Invenio-DB
+-e git+git://github.com/inveniosoftware/invenio-utils#egg=Invenio-Utils
```

Comment:
We should not have a dependency on invenio-utils. Perhaps discuss with @jirikuncar what to do.